<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>523</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>523</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Function Drawer V1.0.2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="draw_button">
        <property name="text">
         <string>RYSUJ</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Liczba punktów na jednostkę:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_resolution">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_liniowa">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>funkcja liniowa:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>                                                    a                                                                           b</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>f(x)=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_liniowa_a">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>x +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_liniowa_b">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>funkcja logarytmiczna</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_logarytmiczna">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>                    a                                        b                                                      c</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>f(x)=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_logA">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_logB">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>x +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_logC"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>funkcja sinusoidalna</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_sinusoidalna">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>                         A                                   f                                     φ                                  B    </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>f(x)= </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sinA">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>sin(2*π*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sinf">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>x+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sinFi"/>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>)+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sinB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="check_pierwiastkowa">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>funkcja pierwiastkowa</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>                              a                                        f                                        d</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>f(x)=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sqrtA">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>sqrt           (</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sqrtX">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>x ) +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_sqrtD"/>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Zakres prezentacji:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_zakY_min">
          <property name="minimum">
           <number>-9999</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>-10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>do:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_zakX_min">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="value">
           <number>-10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_zakY_max">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>do:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>X od:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_zakX_max">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Y od:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEksportuj">
    <property name="title">
     <string>Eksportuj</string>
    </property>
    <widget class="QMenu" name="menuWarto_ci_funkcji_do_pliku">
     <property name="title">
      <string>Wartości funkcji do pliku </string>
     </property>
     <addaction name="actionFunkcji_liniowej"/>
     <addaction name="actionFunkcji_logarytmicznej"/>
     <addaction name="actionFunkcji_sinusoidalnej"/>
     <addaction name="actionFunkcji_pierwiastkowej"/>
    </widget>
    <addaction name="actionEksportuj_wykres_do_bitmapy"/>
    <addaction name="actionWykres_do_PDF"/>
    <addaction name="menuWarto_ci_funkcji_do_pliku"/>
   </widget>
   <widget class="QMenu" name="menuImportuj">
    <property name="title">
     <string>Importuj</string>
    </property>
    <addaction name="actionWarto_ci_funkcji_z_pliku"/>
   </widget>
   <addaction name="menuEksportuj"/>
   <addaction name="menuImportuj"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEksportuj_wykres_do_bitmapy">
   <property name="text">
    <string>Wykres do bitmapy</string>
   </property>
  </action>
  <action name="actionFunkcji_liniowej">
   <property name="text">
    <string>Funkcji liniowej</string>
   </property>
  </action>
  <action name="actionFunkcji_logarytmicznej">
   <property name="text">
    <string>Funkcji logarytmicznej</string>
   </property>
  </action>
  <action name="actionFunkcji_sinusoidalnej">
   <property name="text">
    <string>Funkcji sinusoidalnej</string>
   </property>
  </action>
  <action name="actionFunkcji_pierwiastkowej">
   <property name="text">
    <string>Funkcji pierwiastkowej</string>
   </property>
  </action>
  <action name="actionFunkcja_liniowa">
   <property name="text">
    <string>Funkcja liniowa</string>
   </property>
  </action>
  <action name="actionFunkcja_logarytmiczna">
   <property name="text">
    <string>Funkcja logarytmiczna</string>
   </property>
  </action>
  <action name="actionFunkcja_sinusoidalna">
   <property name="text">
    <string>Funkcja sinusoidalna</string>
   </property>
  </action>
  <action name="actionFunkcja_pierwiastkowa">
   <property name="text">
    <string>Funkcja pierwiastkowa</string>
   </property>
  </action>
  <action name="actionWykres_do_PDF">
   <property name="text">
    <string>Wykres do PDF</string>
   </property>
  </action>
  <action name="actionWarto_ci_funkcji_z_pliku">
   <property name="text">
    <string>Wartości funkcji z pliku</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
